voyager_archive/templates/voyager_archive/marc_display.html

```diff
@@ -11,18 +11,33 @@
 </form>
 <hr/>
 
-{% comment %}
-Raw Marc dump
-{% endcomment %}
+<div class="row">
+    <div class="column">
+        <h2>Record Leader: {{ marc_record.leader }}</h1>
```

Comment:
I suggest dropping this to `h4` (and make start /end tags consistent :) ).  The leader is more critical than the rest of the MARC data.